frontend/src/views/Profile/Profile.tsx

```diff
@@ -42,14 +56,17 @@ export function Profile() {
             setAuctions(await getUsersAuctions(profileID));
             dispatch(fetchUserProducts(profileID));
         })();
```

Comment:
https://dev.to/valeriavg/comment/1f5m2

Do not use raw async functions inside useEffect, declare the function and then invoke